packages/@webex/internal-plugin-metrics/src/call-diagnostic/call-diagnostic-metrics.ts

```diff
@@ -364,6 +366,7 @@ export default class CallDiagnosticMetrics extends StatelessWebexPlugin {
         error = merge(
           {fatal: true, shownToUser: false, name: 'other', category: 'other'}, // default values
           {errorCode: clientErrorCode},
+          {errorData: {errorName: serviceErrorName}},
```

Comment:
we should check that if serviceErrorName is not provided, we don't send 
```
{
  errorData :{
    errorName: undefiend
  }
}
```

Reply:
maybe add a condition after, to add the error.errorData.errorName

Reply:
ok, just checked the tests and indeed it does get sent like that. 
I'm not sure if `clearEmptyKeysRecursively` will remove it or not. Check the network logs, and if you don't see errorData at all, then leave it as is, but if you do see it, then add the condition i mentioned above